indico/htdocs/js/indico/jquery/trackrolewidget.js

```diff
@@ -0,0 +1,94 @@
+/* This file is part of Indico.
```

Comment:
I've been throwing stuff in `.../indico/widget` when the code was strictly related to a WTForm widget. Even if it's a self-contained jQueryUI widget, it's still quite specific and strictly related to the field (expecting certain HTML structure and data available in `data-` attributes), so to me, it feels like it belongs to `.../indico/widget`.